knimin/handlers/ag_edit_participant.py

```diff
@@ -13,6 +13,9 @@ def get(self):
         email = self.get_argument('email', None)
         if email is not None:
             login = db.get_login_by_email(email)
+            if not login:
+                raise ValueError("Couldn't retrieve the login information for "
```

Comment:
Since this is a handler, can it be handled more gracefully? I don't know what will display on the client side for this. If something does, it may be a traceback which will be confusing, or it might not even show a traceback (see qiita's `500` errors)